R/zchunk_L201.en_nonco2.R

```diff
@@ -0,0 +1,358 @@
+      add_title("Steepness of emissions reduction for energy technologies in all regions") %>%
+      add_units("%") %>%
+      add_comments("The steepness is from an assumptions file (A51_steepness). It was chosen to replicate the pollutant reduction rates for SO2 in GCAM3.") %>%
+      add_comments("Steepness is removed for technologies with maximum reduction rates of 0.") %>%
+      add_legacy_name("L201.nonghg_steepness") %>%
+      add_precursors("common/GCAM_region_names",
+                     "emissions/A_regions", "energy/A_regions",
+                     "L151.nonghg_ctrl_R_en_S_T",
+                     "emissions/A51.steepness",
+                     "temp-data-inject/L244.DeleteThermalService") ->
+      L201.nonghg_steepness
+
+    L201.nonghg_max_reduction_res %>%
+      rename(max.reduction = max_reduction) %>% # no idea why old data system renamed this
+      add_title("Maximum reduction for resources in all regions") %>%
+      add_units("%") %>%
+      add_comments("The maximum reduction is calculated in L151 to match the maximum emissions controls assumed in GCAM3.") %>%
+      add_comments("Controls are removed when the maximum reduction is zero.") %>%
+      add_legacy_name("L201.nonghg_max_reduction_res") %>%
+      same_precursors_as(L201.nonghg_max_reduction) ->
+      L201.nonghg_max_reduction_res
+
+    L201.nonghg_steepness_res %>%
+      add_title("Steepness of reduction for resources in all regions") %>%
+      add_units("%") %>%
+      add_comments("The steepness is from an assumptions file (A51_steepness). It was chosen to replicate the pollutant reduction rates for SO2 in GCAM3.") %>%
+      add_comments("Steepness is removed for technologies with maximum reduction rates of 0.") %>%
+      add_legacy_name("L201.nonghg_steepness_res") %>%
+      same_precursors_as(L201.nonghg_steepness) ->
+      L201.nonghg_steepness_res
+
+    L201.nonghg_res %>%
+      add_title("Pollutant emissions for energy resources in all regions") %>%
+      add_units("Tg/EJ") %>%
+      add_comments("Take non-GHG emissions for the energy system, filter to include only resources") %>%
+      add_comments("in model base years, rename to regional SO2, filter to district heat,") %>%
+      add_comments("and delete sectors with zero heating and zero cooling degree days.") %>%
+      add_legacy_name("L201.nonghg_res") %>%
+      add_precursors("common/GCAM_region_names",
+                     "emissions/A_regions",
+                     "temp-data-inject/L111.nonghg_tgej_R_en_S_F_Yh") ->
+      L201.nonghg_res
+
+    L201.ghg_res %>%
+      add_title("GHG emissions from resource production in all regions") %>%
+      add_units("Tg/EJ") %>%
+      add_comments("Take GHG emissions for the energy system, filter to include only resources") %>%
+      add_comments("in model base years, rename to regional SO2, filter to district heat,") %>%
+      add_comments("and delete sectors with zero heating and zero cooling degree days.") %>%
```

Comment:
I just noticed this. We don't actually have to filter out district heat or delete sectors with zero HDD/CDD for the two resource files (`L201.nonghg_res` and `L201.ghg_res`) since those aren't resources. Maybe adjust those comments. Otherwise I think it is good.